lib/spaces/framework/streaming/producing.rb

```diff
@@ -4,8 +4,11 @@ module Producing
     def produce(&block)
       yield(self)
     rescue => e
-      logger.error(e)
-      exception(e)
+      if command.input[:background]
```

Comment:
I'd prefer a ternary here, and I'd probably change it if I swung by this code later, but it's no biggy.